apps/backoffice-v2/src/pages/BusinessesAlertsAnalysis/components/OngoingMonitoringTable/components/DownloadReportButton/DownloadReportButton.tsx

```diff
@@ -0,0 +1,22 @@
+import { Button } from '@/common/components/atoms/Button/Button';
+import { useDownloadReportButtonLogic } from '@/pages/BusinessesAlertsAnalysis/components/OngoingMonitoringTable/components/DownloadReportButton/useDownloadReportButtonLogic';
+import { Download } from 'lucide-react';
+import { FunctionComponent } from 'react';
+
+interface IDownloadReportButtonProps {
+  fileId: string;
+  reportId: string;
+}
+
+export const DownloadReportButton: FunctionComponent<IDownloadReportButtonProps> = ({
+  fileId,
+  reportId,
+}) => {
+  const { isDownloading, downloadReport } = useDownloadReportButtonLogic({ fileId, reportId });
+
+  return (
+    <Button variant="ghost" size="icon" disabled={isDownloading} onClick={downloadReport}>
+      <Download color="#007AFF" />
+    </Button>
+  );
+};
```

Comment:
1. We already have the URL to the report - we can achieve this with a link with the `download` attribute
2. We prefer having a link that opens the PDF in a new tab - the native viewer also has a download and print button